sandbox/controls/Buttons.qml

```diff
@@ -343,4 +343,26 @@ Column {
         text: "Button with Emoji"
         icon.emoji: "🖼️️"
     }
+
+    RowLayout {
+        spacing: 20
+
+        StatusIconTextButton {
+            Layout.alignment: Qt.AlignVCenter
+            spacing: 0
+            statusIcon: "next"
+            icon.width: 24
+            icon.height: 24
+            iconRotation: 180
+            text: "Previous page"
+            font.pixelSize: 15
+            onClicked: testText.visible = !testText.visible
+        }
```

Comment:
How exactly is this different from `StatusFlatButton` that has an `icon`

Reply:
It doesn't have an hover color, just only like a text link. Yeah.. I'm thinking maybe it is redundant and the way to go for it is by adding a variant in `StatusFlatButton`... WDYT?

Reply:
Not sure from the top of my head, but maybe we expose the `sensor` in `StatusFlatButton`, then you could simply make it not have a hover (although, it sounds backwards to me to introduce/have a control that looks like a link/or button but doesn't have a hover indication. How will users know they can click it?)

Reply:
I have a list of design questions for the task I'm working on and one of them is this, so I'll merge this PR as it is and move forward but once I've the response from design, I'll update `StatusQ` accordingly (It doesn't make sense to do more changes here if at the end the design team decides that it also needs hover color.. so then I'll remove the `statusIconTextButton` added here).

BTW, this is an example of the `link` button from a design:
![Screenshot 2022-07-13 at 13 10 46](https://user-images.githubusercontent.com/97019400/178721172-b1142afb-2122-413d-bff8-ed2052060958.png)